Assets/Plugins/GameJolt/Scripts/UI/Controllers/SignInWindow.cs

```diff
@@ -8,21 +8,26 @@ public class SignInWindow: BaseWindow
 		public InputField usernameField;
 		public InputField tokenField;
 		public Text errorMessage;
+		public Toggle rememberMeToggle;
 
 		Action<bool> signedInCallback;
 		Action<bool> userFetchedCallback;
 
 		public override void Show(Action<bool> callback)
 		{
-			Show(callback, null);
+			Show(callback, null, false);
 		}
 
-		public void Show(Action<bool> signedInCallback, Action<bool> userFetchedCallback)
-		{
+		public void Show(Action<bool> signedInCallback, Action<bool> userFetchedCallback, bool autoLogin) {
 			errorMessage.enabled = false;
 			animator.SetTrigger("SignIn");
 			this.signedInCallback = signedInCallback;
 			this.userFetchedCallback = userFetchedCallback;
+			string username, token;
+			rememberMeToggle.isOn = API.Manager.Instance.GetStoredUserCredentials(out username, out token);
+			usernameField.text = username;
+			tokenField.text = token;
+			if(rememberMeToggle.isOn && autoLogin) Submit();
```

Comment:
Does this makes the window to show and immediately hide?

Reply:
the window will close when we get the answer from the gamejolt server. So typically it will close *nearly* immediately